hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/SparkUpsertCommitActionExecutor.java

```diff
@@ -44,6 +44,7 @@ public SparkUpsertCommitActionExecutor(HoodieSparkEngineContext context,
   @Override
   public HoodieWriteMetadata<JavaRDD<WriteStatus>> execute() {
     return SparkWriteHelper.newInstance().write(instantTime, inputRecordsRDD, context, table,
-        config.shouldCombineBeforeUpsert(), config.getUpsertShuffleParallelism(), this, true);
+        config.shouldCombineBeforeUpsert(), config.getUpsertShuffleParallelism(),
+            this, Boolean.parseBoolean(config.getProps().getProperty("hoodie.tagging.before.insert", "true")));
```

Comment:
would you move `hoodie.tagging.before.insert` and default value to HoodieWriteConfig?